src/smk/viewer-leaflet/tool/minimap/tool-minimap-leaflet.js

```diff
@@ -6,7 +6,7 @@ include.module( 'tool-minimap-leaflet', [ 'leaflet', 'tool-minimap' ], function
 
         smk.addToStatus( $( '<div class="smk-spacer">' ).height( 170 ).get( 0 ) )
 
-        var ly = smk.$viewer.createBasemapLayer( this.baseMap || "Topographic" );
+        var ly = smk.$viewer.createBasemapLayer( smk.viewer.esriApiKey, this.baseMap || "Topographic" );
```

Comment:
Should we make the default StamenTonerLight?

Reply:
I think we should keep Topographic the default - see my reply on the review comment.

Reply:
My concern with using an Esri basemap is that apps appear broken if you haven't changed the basemap to StamenTonerLight or added configured an esriApiKey. This doesn't make for a great first experience for new users. Perhaps worth discussing with Nicole?

Reply:
Ok, I still don't like the idea of delivering a product that appears broken out of the box, but I'll stand aside and protest silently!